docs/install/knative-with-operators.md

```diff
@@ -0,0 +1,244 @@
+making progress in extending their capabilities. One of the major missing points is that Knative operators do not currently
+provide HA capabilities.
+
+The open source versions of Knative operators have not been tested on any existing production environment, so use
+them for development or test purpose. However, we encourage you to experiment Knative operators on top of the Kubernetes
+cluster for your production environment, helping us find out what we can potentially improve in Knative operators by
+reporting your issues.
+
+## Install Knative Serving with Operator:
+
+All the releases for Knative Serving Operator can be found on [this page](https://github.com/knative/serving-operator/releases).
+
+### From releases:
+
+Replace \<version\> with the latest version or the version you would like to install, and run the following command to
+install Knative Serving Operator:
+
+```
+kubectl apply -f https://github.com/knative/serving-operator/releases/download/<version>/serving-operator.yaml
+```
+
+### From source code:
+
+The source code is available on this page. You need to install the building tool [ko](https://github.com/google/ko) first. After downloading the
+source code, you can install the operator in the root directory of the source code with the following command:
+
+```
+ko apply -f config/
+```
+
+Verify the installation of Knative Serving Operator with the command:
+
+```
+kubectl get deployment knative-serving-operator
+```
+
+You should see the deployment is ready, if the operator is installed.
+
+Use the following command to track the log of the operator:
+
+```
+kubectl logs -f $(kubectl get pods -l name=knative-serving-operator -o name)
+```
+
+If everything goes fine, it is time to install Knative Serving by installing the custom resource of Knative Serving
+operator. Operator supports to install Knative Serving under any namespace, which needs to be created as well. Then,
+we can create a custom resource with empty spec section. Technically, you can use any namespace, but we recommend you
+to use `knative-serving`. In the rest of this tutorial, we keep on using `knative-serving` as the namespace to create the
+Serving operator CR and all the other namespaced Serving resources. To create the CR, you can run the following command:
+
+```
+cat <<-EOF | kubectl apply -f -
+apiVersion: v1
+kind: Namespace
+metadata:
+ name: knative-serving
+---
+apiVersion: operator.knative.dev/v1alpha1
+kind: KnativeServing
+metadata:
+  name: knative-serving
+  namespace: knative-serving
+EOF
+```
+
+Wait some time for Knative Serving is to become ready. Check the Knative Serving deployment with the following command:
+
+```
+kubectl get deployment -n knative-serving
+```
+
+You will receive the status of Knative serving deployments. If Knative Serving has been successfully deployed, you will
+see all the deployments of the Knative Serving have reached READY status.
+
+## Install Knative Eventing with Operator:
```

Comment:
Rewrite as per the comments for Serving - I think all of the same changes should apply.